gatsby_fp/package.json

```diff
@@ -5,18 +5,33 @@
   "version": "0.1.0",
   "author": "Kyle Mathews <mathews.kyle@gmail.com>",
   "dependencies": {
+    "axios": "^0.19.2",
+    "db": "^2.0.0",
+    "dotenv": "^8.2.0",
+    "express": "^4.17.1",
+    "express-graphql": "^0.9.0",
     "gatsby": "^2.20.12",
     "gatsby-image": "^2.3.1",
     "gatsby-plugin-manifest": "^2.3.3",
     "gatsby-plugin-offline": "^3.1.2",
     "gatsby-plugin-react-helmet": "^3.2.1",
     "gatsby-plugin-sharp": "^2.5.3",
+    "gatsby-source-apiserver": "^2.1.4",
     "gatsby-source-filesystem": "^2.2.2",
+    "gatsby-source-multi-api-FORKED": "https://github.com/Stevens-EWS/gatsby-source-multi-api",
+    "gatsby-source-rest-api": "^0.2.3",
     "gatsby-transformer-sharp": "^2.4.3",
+    "graphiql": "^0.17.5",
+    "json-server": "^0.16.1",
+    "minify": "^5.1.1",
+    "nodemon": "^2.0.3",
```

Comment:
`nodemon` should be under devDependencies not under dependencies. We're not going to use it in production so there's no need to put it here.